src/async_impl/client.rs

```diff
@@ -626,6 +629,19 @@ impl ClientBuilder {
         self
     }
 
+    /// Set that all sockets have `SO_KEEPALIVE` set with the supplied duration.
+    ///
+    /// If `None`, the option will not be set.
+    ///
+    /// Default is 60 seconds.
+    pub fn tcp_keepalive<D>(mut self, val: D) -> ClientBuilder
```

Comment:
Looks good, mind just moving this down to the `// TCP Options` section, since the order of methods affects the order shown in the generated docs.